corehq/apps/app_manager/templates/app_manager/form_summary_base.html

```diff
@@ -65,7 +65,10 @@ <h5>
                 <ol data-bind="foreach: questions">
                     <li data-bind="visible: isVisible">
                         <i data-bind="attr: { 'class': $root.questionIcon($data), 'title': type }"></i>
+                        <!-- ko ifnot: $root.readOnly -->
```

Comment:
root.readOnly is only true for linked apps?

Reply:
currently, yes: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views/app_summary.py#L56